go/vt/mysqlctl/metadata_tables.go

```diff
@@ -46,6 +46,9 @@ var (
 	sqlAlterLocalMetadataTable = []string{
 		`ALTER TABLE _vt.local_metadata ADD COLUMN db_name VARBINARY(255) NOT NULL DEFAULT ''`,
 		`ALTER TABLE _vt.local_metadata DROP PRIMARY KEY, ADD PRIMARY KEY(name, db_name)`,
+		// VARCHAR(255) is not long enough to hold replication positions, hence changing to
+		// MEDIUMBLOB.
+		`ALTER TABLE _vt.local_metadata CHANGE value value MEDIUMBLOB`,
```

Comment:
If this is run again, does it result in the same `ERDupFieldName` error that we already ignore?

Also, do we need to add `NOT NULL` to match the CREATE TABLE version?

Reply:
Running again produces no errors. 
We don't need to add `NOT NULL`. Changing the column type doesn't change it's nullable status.